kibana-reports/public/components/app.tsx

```diff
@@ -55,10 +55,22 @@ interface OpendistroKibanaReportsAppDeps {
 
 const styles: CSS.Properties = {
   float: 'left',
-  width: "100%",
+  width: '100%',
   maxWidth: '1600px',
 };
 
+export const permissionsMissingToast = (action: string) => {
```

Comment:
Why add this helper function in `app.tx`, I'd suggest create a file under utils folder

Reply:
Just added it here so it's easily accessible by all files since the permissions errors can happen in many places- but good idea